src/icons/IconTuneVarticalVariant.tsx

```diff
@@ -0,0 +1,14 @@
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+export const IconTuneVarticalVariant = (props: SvgIconProps) => {
```

Comment:
```suggestion
export const IconTuneVerticalVariant = (props: SvgIconProps) => {
```
И в названии файла ошибка